tests/test_plotting.py

```diff
@@ -45,8 +50,22 @@ def domains_data() -> PlotEventData:
 def fig(request) -> plt.figure:
     """Creates the fixture for the tests."""
     plt.close("all")
-    figure = plt.subplots(1, 3)[0]
-    return plot_ref_sld_helper(fig=figure, data=domains_data() if request.param else data())
+    figure = plt.subplots(1, 2)[0]
+    return RATplot.plot_ref_sld_helper(fig=figure, data=domains_data() if request.param else data())
```

Comment:
oh right, i assumed it was a typo. for that exact reason i think that it should be its own test rather than implicit so that we can tell that we're actually testing it!